.github/workflows/test.yml

```diff
@@ -115,8 +125,8 @@ jobs:
         if: ${{ matrix.suite.name != 'GitHub Integration' || matrix.os == 'ubuntu' }}
         run: go test github.com/jfrog/frogbot/v2/${{ matrix.suite.package }} -v -race -timeout 30m -cover
         env:
-          JF_URL: ${{ secrets.PLATFORM_URL }}
-          JF_ACCESS_TOKEN: ${{ secrets.PLATFORM_ADMIN_TOKEN }}
+          JF_URL: ${{ github.event_name == 'workflow_dispatch' && secrets.DEV_MASTER_PLATFORM_URL || secrets.PLATFORM_URL }}
+          JF_ACCESS_TOKEN: ${{ github.event_name == 'workflow_dispatch' && secrets.DEV_MASTER_PLATFORM_ADMIN_TOKEN || secrets.PLATFORM_ADMIN_TOKEN }}
```

Comment:
you can do it once using for all tests with something like this:

```
jobs:

  job1:
    steps:
      - name: set envs
        run: echo "JF_URL=${{ github.event_name == 'workflow_dispatch' && secrets.DEV_MASTER_PLATFORM_ADMIN_TOKEN || secrets.PLATFORM_ADMIN_TOKEN }}" >> $GITHUB_ENV

  job2:
    env:
      JF_URL: ${{ env.JF_URL }}
    steps:

  job3:
    env:
      JF_URL: ${{ env.JF_URL }}
    steps:
```

just let @yahavi see if it's secure enough

Reply:
One potential issue I notice with this approach is that the new access token environment variable won't be considered confidential, but we can address it by concealing it in the logs: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#masking-a-value-in-a-log

Reply:
after many tries this unification is very risky, so the credentials resolution will stay as is